lib/tinyMCE/extlink-tinymce.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+  tinymce.PluginManager.add( 'extlinks', function( editor, url ) {
+
+    // Add Button to Visual Editor Toolbar
+    editor.addButton('extlinks', {
+      title: 'Make Links External',
+      cmd: 'extlinks'
+    });
+
+    jQuery('.mce-i-extlinks').addClass('dashicons-before dashicons-external');
+
+    // Add Command when Button Clicked
+    editor.addCommand('extlinks', function() {
+
+      // Check we have selected some text selected
+      var text = editor.getContent({
+        'format': 'html'
+      });
+
+      text = text.replace(/ href=/g, ' target="_blank" href=');
```

Comment:
apparently tinymce or some magic involved fixes the doubled target when added 

@varffs what about internal links?
i'm worried this button will lead to links that open in new tabs all over the site without reason and i think that sux